radixdlt/src/integration/java/com/radixdlt/store/berkeley/BerkeleySearchCursorTests.java

```diff
@@ -40,7 +41,9 @@ public void store_single_atom__search_by_unique_aid_and_get() throws Exception {
 
 		List<LedgerEntry> ledgerEntries = ledgerEntryGenerator.createLedgerEntries(identity, 1);
 		StoreIndex uniqueIndex = new StoreIndex(PREFIX, ledgerEntries.get(0).getAID().getBytes());
-		getStore().commit(ledgerEntries.get(0), ImmutableSet.of(uniqueIndex), ImmutableSet.of());
+		Transaction tx = getStore().createTransaction();

```

Comment:
I think this should really look like this:
```
Transaction tx = getStore().createTransaction();
try {
    getStore().execute(tx, ledgerEntries.get(0), ImmutableSet.of(uniqueIndex), ImmutableSet.of());
    tx.commit();
} finally {
    tx.rollbackIfStillOpen();
}
```

...or at least:
```
Transaction tx = getStore().createTransaction();
try {
    getStore().execute(tx, ledgerEntries.get(0), ImmutableSet.of(uniqueIndex), ImmutableSet.of());
    tx.commit();
} catch (Exception e) {
    tx.rollback();
}
```

Perhaps even better would be to abstract this out altogether:
```
getStore().withTransaction((store, tx) -> {
    store.execute(tx, ledgerEntries.get(0), ImmutableSet.of(uniqueIndex), ImmutableSet.of());
});
```

This really applies to all the spots where transactions are used.  Appreciate this is test code, but given that it's a prime copy/pasta candidate, it's worth doing right.  Also leaving transactions open may impact downstream tests.